apps/rstudio/Dockerfile

```
@@ -0,0 +1,48 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:24.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+ARG APP_NAME
+ARG APP_VERSION
+
+WORKDIR /apps/${APP_NAME}
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    curl wget gpg gpg-agent software-properties-common \
+    build-essential cmake dirmngr file libssl-dev libpq5 \
+    libclang-dev libnss3 libatk1.0-0 libatk-bridge2.0-0 \
+    libcups2 libgtk-3-0 libasound2t64 gfortran liblapack-dev \
+    libopenblas-dev libpng-dev libxkbcommon-x11-0 && \
+    wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | gpg --dearmor > cran-archive-keyring.gpg && \
+    install -D -o root -g root -m 644 cran-archive-keyring.gpg /etc/apt/keyrings/cran-archive-keyring.gpg && \
```

what's wrong with it?
```suggestion
    curl https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | gpg --dearmor -o /etc/apt/keyrings/cran-archive-keyring.gpg && \
```

---

Probably nothing but i used the vscode example as you advised and it's probably overkill and can be simplified , testing now

---

Steak is cooked, nothing wrong with that